aptos-move/framework/src/natives/cryptography/algebra/mod.rs

```diff
@@ -289,7 +287,6 @@ fn deserialize_internal(
 
 macro_rules! ark_add_internal {
     ($gas_params:expr, $context:expr, $args:ident, $structure:expr, $ark_type:ty) => {{
-        $context.charge($gas_params.placeholder)?;
```

Comment:
Do you wanna add a todo comment here so we do not forget adding it back?

Reply:
We don't have to. The full PR should fix everything.

Reply:
Can you add a note in the full PR?